jc/parsers/xrandr.py

```diff
@@ -189,15 +189,26 @@
       ]
     }
 """
+from collections import defaultdict
+from enum import Enum
 import re
-from typing import Dict, List, Optional, Union
+from typing import Dict, List, Tuple, Union
+
 import jc.utils
 from jc.parsers.pyedid.edid import Edid
 from jc.parsers.pyedid.helpers.edid_helper import EdidHelper
 
+Match = None
+try:
+    # Added Python 3.7
+    Match = re.Match
+except AttributeError:
+    Match = type(re.match("", ""))
+
 
 class info:
     """Provides parser metadata (version, author, etc.)"""
+
     version = "1.4"
```

Comment:
Nice refactor! Let's update the parser version to 2.0. Looks like tests are passing fine. I can get this in the v1.25.1 release.

Reply:
Thanks! Version bumped.